### ISSUE TITLE
Experimental Results on Gowalla dataset

### ISSUE THREAD

**Issue by trannhuthuat96** [openbenchmark/BARS#15]
Hello,

Thank you so much for your very useful work.

I have a question about results on Gowalla dataset, based on the log in https://github.com/openbenchmark/BARS/tree/master/candidate_matching/benchmarks/SimpleX/SimpleX_gowalla_x1

How do you divide training data into training and validation sets? From the log, test set is currently treated as validation set and reported results are selected as the best iteration on test set, which causes test data leakage. A similar problem was reported in this paper https://arxiv.org/pdf/2005.09683.pdf?

Look forward to hearing from you soon,
Thank you so much.

**Comment by zhujiem** [comment]
Yes, we follow the original data split with only train/test subsets and thus the results can be seen as validation accuracy.

**Comment by trannhuthuat96** [comment]
Got it, thanks!
